src/core/js/models/adaptModel.js

```diff
@@ -236,9 +245,9 @@ define([
         },
 
         getAvailableChildren: function() {
-            return new Backbone.Collection(this.getChildren().where({
+            return this.getChildren().where({
```

Comment:
``duplicate functions, depr warning, remove on v4, give time to update, w. few acute breaking issues``

^ we all decided this in the gitter room, after a brief conversation about the merits of dropping vs deprecating